source/common/common/utility.cc

```diff
@@ -67,6 +69,55 @@ bool StringUtil::atoul(const char* str, uint64_t& out, int base) {
   }
 }
 
+absl::string_view StringUtil::ltrim(absl::string_view source) {
+  source.remove_prefix(std::min(source.find_first_not_of(" \t\f\v\n\r"), source.size()));
+  return source;
+}
+
+absl::string_view StringUtil::rtrim(absl::string_view source) {
+  source.remove_suffix(source.size() - source.find_last_not_of(" \t\f\v\n\r") - 1);
+  return source;
+}
+
+absl::string_view StringUtil::trim(absl::string_view source) { return ltrim(rtrim(source)); }
+
+bool StringUtil::find(absl::string_view source, absl::string_view delimiters,
```

Comment:
can you rename this findToken?  I don't think the overload of find() makes sense.